tempest/run-tempest.sh

```diff
@@ -57,6 +59,16 @@ export NOSE_WITH_XUNIT=1
 export NOSE_XUNIT_FILE=tempest_xunit.xml
 
 cd /usr/share/openstack-tempest-juno/
+
+# Quick check using Javelin
+if javelin_check_resources; then
```

Comment:
Yes, you're right. I wanted to follow what's currently described in the [docs on Sanity](http://spinalstack.enovance.com/en/latest/deploy/sanity.html).

>  Before creatind the resources, it checks if they already exist. If it’s the case, it considers that Sanity already built before. This is useful when running sanity after the upgrade process.

The only way I can think of doing an actual check is either:

1. Have 2 separate jobs, sanity-pre-upgrade and sanity-post-upgrade.
2. Include Javelin testing as part of the upgrade job, so the job is only marked as success if Javelin testing succeeds.

Which one do you think is best? Or maybe you can think of a better way?